blocks/header-nav-chain-block/chains/header-nav-chain-block/default.jsx

```diff
@@ -133,6 +133,7 @@ const Nav = (props) => {
     desktopNavivationStartHeight,
     shrinkDesktopNavivationHeight,
     showHorizontalSeperatorDots,
+    ariaLabel = 'Sections Menu',
```

Comment:
Default set here and in `defaultProps` we should on need to set it in one place

Reply:
Okay this will be updated to just be here in the props